apps/web/pages/event-types/index.tsx

```diff
@@ -606,13 +585,6 @@ const EventTypeListHeading = ({
           </Link>
         )}
       </div>
-      {!profile?.slug && !!teamId && (
-        <button onClick={() => publishTeamMutation.mutate({ teamId })}>
-          <Badge variant="gray" className="mb-1 -ml-2">
-            {t("upgrade")}
-          </Badge>
-        </button>
-      )}
```

Comment:
Is this change intended for this PR¡

Reply:
no not from me

Reply:
https://github.com/calcom/cal.com/pull/6936/files#r1116335299